DependencyInjection/DoctrineExtension.php

```diff
@@ -760,6 +762,23 @@ private function loadPropertyInfoExtractor($entityManagerName, ContainerBuilder
         $propertyExtractorDefinition->addTag('property_info.type_extractor', ['priority' => -999]);
     }
 
+    /**
+     * Loads a validator loader for each defined entity manager.
+     *
+     * @param string $entityManagerName
+     */
+    private function loadValidatorLoader($entityManagerName, ContainerBuilder $container)
```

Comment:
This package requires php 7.1, I think you can use type hints here:

```suggestion
    private function loadValidatorLoader(string: $entityManagerName, ContainerBuilder $container): void
```